api/swagger.yaml

```diff
@@ -130,7 +130,7 @@ paths:
         "404":
           description: App not found
       summary: Update 1 or more versions of an app
-  /apps/:id/versions/batch-disable:
+  /apps/:id/versions/disable:
```

Comment:
It is missing update the tests which call the endpoint

Reply:
Probably a separate PR as a new JIRA as this is just fixing the Swagger doc.

Reply:
I will do it here for the sake of time though.